src/utils/getInitialURL.js

```diff
@@ -0,0 +1,46 @@
+export const parseUrl = () => {
+  const url = window.location.href
+  const geocoordmatch = url.substring(url.indexOf('@'))
+  console.log('match', geocoordmatch)
+
+  const getCoords = isValidCoord(geocoordmatch)
+
+  if (getCoords.valid) {
+    return {
+      center: { lat: getCoords.lat, lng: getCoords.lng },
+      zoom: getCoords.zoom,
+    }
+  } else {
+    return { center: { lat: 40.1125785, lng: -88.2287926 }, zoom: 1 }
```

Comment:
What is the significance of this coordinate? Also, if we are hard coding coordinates, assign a variable to these values like:
`DEFAULT_LAT = 40.1125785` to it, but change the variable name to whatever it represents